PhysicsTools/PatAlgos/plugins/PATElectronSlimmer.cc

```diff
@@ -136,6 +140,20 @@ pat::PATElectronSlimmer::produce(edm::Event & iEvent, const edm::EventSetup & iS
         if (dropSaturation_(electron)) { electron.setSaturationInfo(reco::GsfElectron::SaturationInfo()); }
         if (dropExtrapolations_(electron)) { electron.setTrackExtrapolations(reco::GsfElectron::TrackExtrapolations());  }
         if (dropClassifications_(electron)) { electron.setClassificationVariables(reco::GsfElectron::ClassificationVariables()); electron.setClassification(reco::GsfElectron::Classification()); }
+
+        if(computeMiniIso_){
+            pat::MiniIsolation miniiso;
+            // veto on candidates in deadcone only in endcap
+            if(fabs(electron.p4().eta()) > 1.479)
+                miniiso = pat::GetMiniPFIsolation(pc.product(), electron.p4(), 0.05, 0.2, 10., 0.0, 0.015, 0.015, 0.08, 0.);
+            else
+                miniiso = pat::GetMiniPFIsolation(pc.product(), electron.p4(), 0.05, 0.2, 10., 0.0, 0., 0., 0., 0.);
```

Comment:
these magic numbers need some documentation and should also probably be made configurable.